packages/mui-lab/src/DesktopDatePicker/DesktopDatePicker.tsx

```diff
@@ -60,6 +60,7 @@ const DesktopDatePicker = React.forwardRef(function DesktopDatePicker<TDate>(
 
   return (
     <DesktopWrapper
+      {...other}
```

Comment:
Should we explicitly add the props we are intending to add? There are other props in the `other` which were not being passed before, so we may break something.